elements/stagger.lua

```diff
@@ -33,19 +33,7 @@ The default StatusBar texture will be applied if the UI widget doesn't have a st
 local parent, ns = ...
 local oUF = ns.oUF
 
--- percentages at which the bar should change color
-local STAGGER_YELLOW_TRANSITION = STAGGER_YELLOW_TRANSITION
-local STAGGER_RED_TRANSITION = STAGGER_RED_TRANSITION
-
--- table indices of bar colors
-local STAGGER_GREEN_INDEX = STAGGER_GREEN_INDEX or 1
-local STAGGER_YELLOW_INDEX = STAGGER_YELLOW_INDEX or 2
-local STAGGER_RED_INDEX = STAGGER_RED_INDEX or 3
```

Comment:
We usually keep these around incase Blizzard decides to remove them. It's happened a couple of times before, so better safe than sorry.